### ISSUE TITLE
top_k 

### ISSUE THREAD

**Issue by xderzzh** [00why00/Glocal#11]
Hello, I find "For the setting of hyperparameters, we set T in Eq. 9 to the k value in the topk function.“ in your paper. But when I look at the code I find that the default value for top_k is 1 and the default value for temperature is 64. Could you please tell me the settings of these two parameters during the experiment? Thank you!

**Comment by 00why00** [comment]
Sorry for the late response.

Eq. 9 corresponds to the following lines of code. We have set T to the k value in the topk function. And the temperature in argparse is the temperature over the logits. 

https://github.com/00why00/Glocal/blob/db161dbf1e80fabfccd96cd323bf485e50f70e1c/model/models/glocal_energy.py#L98-L100

As stated in the paper, the choice of topk value had little effect on the results. And the topk value of minImageNet 1/5-shot, TieredImagenet 1/5 shot and CIFAR-FS 1/5-shot are 10, 1, 2, 5, 1, 1, respectively.
